src/components/dataTable.js

```diff
@@ -224,27 +224,25 @@
     const where = useFilter(completeFilter);
 
     // TODO: move model to skip
-    const { loading, error, data, refetch } =
-      model &&
-      useAllQuery(model, {
-        rawFilter: where,
-        variables,
-        skip: loadOnScroll ? skip : page * rowsPerPage,
-        take: loadOnScroll ? autoLoadTakeAmountNum : rowsPerPage,
-        onCompleted(res) {
-          const hasResult = res && res.results && res.results.length > 0;
-          if (hasResult) {
-            B.triggerEvent('onSuccess', res.results);
-          } else {
-            B.triggerEvent('onNoResults');
-          }
-        },
-        onError(err) {
-          if (!displayError) {
-            B.triggerEvent('onError', err);
-          }
-        },
-      });
+    const { loading, error, data, refetch } = useAllQuery(model, {
+      rawFilter: where,
+      variables,
+      skip: !model || loadOnScroll ? skip : page * rowsPerPage,
```

Comment:
skip: loadOnScroll ? skip : (page ? (page - 1) * rowsPerPage : 0),

Else the filter interaction does not show the record when on page 1

Reply:
Hey @vdberganton thanks, I'll make that change :)